test/conformance/kuttl/kuttl-test.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: kuttl.dev/v1beta1
```

Comment:
I only see `kuttle-test.yaml` for generate tests. Do we need this for other tests?

Reply:
I'm still doing a re-org on this. Any comment on the directory structure? Does it work for you?